### PR TITLE
🔨: Add script to clean up build artifacts and caches (macOS/Windows)

### PR DESCRIPTION
**Pull request by in-a-paddy**
## やったこと

- [x] Add runner to run tasks on not only macOS but Windows
- [x] Add npm script to cleanup build artifacts and caches
- [x] Add script usage to README.md

## やっていないこと

- Test with Git Bash of Git for WIndows
  - I don't have an environment

## 動作確認

- [x] Run `npm run reset-cache` on macOS
- [x] Run `npm run reset-cache -- -i` on macOS
- [x] Run `npm run reset-cache` on Windows PowerShell
- [x] Run `npm run reset-cache -- -i` on Windows PowerShell
- [x] Run `npm run reset-cache` on Windows command prompt
- [x] Run `npm run reset-cache -- -i` on Windows command prompt

## デバイス

It's not about the devices.

## その他（レビュアーへの申し送り事項、懸念点など）

None

### CODE REVIEW

File: .script/runner.js [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+  const tasks = require(`./jobs/${job}-${platform}`);
+  // TODO: Check if task is found
+
+  if (args.includes('--interactive') || args.includes('-i')) {
+    const cui = createInterface({
+      input: process.stdin,
+      output: process.stdout,
+    });
+
+    for (task of tasks) {
```
Comment (Takumon) [inline .script/runner.js:60]: letのつけ忘れですか?

**Review by Takumon — CHANGES_REQUESTED** [review]
Please check a review comment.

File: .script/runner.js (hunk 1 above)
Comment (in-a-paddy) [reply .script/runner.js]: つけ忘れでした、、、

**Review by Takumon — APPROVED** [review]
LGTM✨

**Pull request merged**